src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -178,43 +178,62 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
   }
 
   fetchCounts = async () => {
-    const { allowedTypes } = this.props;
+    const { allowedTypes, namespaceRegistry } = this.props;
     const { queryText, visibleTypes, visibleNamespaces } = parseQuery(this.state.activeQuery);
 
     const filteredTypes = allowedTypes.filter(
       (type) => !visibleTypes || visibleTypes.includes(type)
     );
 
+    let availableNamespaces = namespaceRegistry.getAll() || [];
+    availableNamespaces = availableNamespaces.map((ns) => ns.id);
+
+    const filteredCountOptions = {
+      typesToInclude: filteredTypes,
+      searchString: queryText,
+    };
+
+    if (availableNamespaces?.length) {
```

Comment:
189 assumes `availableNamespaces` is an Array so the conditional for `availableNamespaces?.length` is unnecessary. But is `availableNamespaces` an Array always?

Reply:
Yes it will always be an array. See https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/saved_objects_management/public/services/namespace_service.ts#L26.

Reply:
Are you suggesting that the condition can be shortened to `if (availableNamespaces) {...`?

This condition exists to conditionally add `namespacesToInclude` to the query depending if there are namespaces registered.

Reply:
Since it will always be an array, `availableNamespaces?.length` should just be `availableNamespaces.length`, right? The `?` is pointless.

Reply:
That's right. Updated all instances of this for conciseness.